rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/CommandSet.java

```diff
@@ -707,6 +708,17 @@
                                                                          .commandClass(EditNodeSourceCommand.class)
                                                                          .entry();
 
+    public static final CommandSet.Entry NS_UPDATE_PARAM = CommandSetEntryBuilder.newInstance()
+                                                                                 .opt("upns")
+                                                                                 .longOpt("updatensparam")
+                                                                                 .description("Update the dynamic parameters of a node source")
+                                                                                 .hasArgs(true)
+                                                                                 .numOfArgs(1)
+                                                                                 .argNames("node-source")
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "node-source" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)